MCU_Aurix/driver/inc/SCU_Reset.h

```diff
@@ -0,0 +1,35 @@
+/**********************************************************************************************************************
```

Comment:
Is this file + class separately needed? Since this header is included in MCU_Aurix/driver/src/reset.c, the Infineon headers and types are indirectly included. If it should be completely separated, we need to think about a different solution